templates/rank.html

```diff
@@ -0,0 +1,100 @@
+                                <option value="master">大神模式</option>
+                            </select>
+                        </div>
+                        <div class="col selectform">
+                            <select name="game_body">
+                                <option value="hands">手</option>
+                                <option value="elbow">手肘</option>
+                                <option value="foot">腳</option>
+                                <option value="head">頭</option>
+                            </select>
+                        </div>
+                        <div class="col selectform">
+                            <select name="game_level">
+                                <option value="1">Level 1</option>
+                                <option value="2">Level 2</option>
+                                <option value="3">Level 3</option>
+                                <option value="4">Level 4</option>
+                                <option value="5">Level 5</option>
+                                <option value="6">Level 6</option>
+                                <option value="7">Level 7</option>
+                                <option value="8">Level 8</option>
+                            </select>
+                        </div>
+                        <div class="col">
+                            <input type="submit" class="clkbutton sub-btn" value="submit"/>
+                        </div>
+                        <div class="col-2"></div>
+                    </div>
+                </form>    
+                <br>
+                <div class="">
+                    <table class="tcontainer">
+                        <thead>
+                        <tr>
+                            <th>排名</th>
+                            <th>姓名</th>
+                            <th>遊戲模式</th>
+                            <th>身體部位</th>
+                            <th>關卡</th>
+                            <th>秒數</th>
+                        </tr>
+                        </thead>
+                        <tbody> 
+                        {% for row in data %}
+                        <tr>
+                            <td>{{loop.index}}</td>
+                            <td>{{ row[1]}}</td>
+                            <td>{{ row[2] }}</td>
+                            <td>{{ row[3] }}</td>
+                            <td>{{ row[4] }}</td>
+                            <td>{{ row[5] }}</td>    
+                        </tr>
+                        </tbody>
+                        {% endfor %}
+                    </table>
+                </div>
+                <div class="stars"></div>
+                <div class="stars1"></div>
+                <div class="stars2"></div>
+                <div class="shooting-stars"></div>
+            </div>
+            <button type="button" class="clkbutton back-btn" role="button" 
+            onclick="javascript:location.href='/'">back</button>
```

Comment:
Rewrite like this:

https://github.com/asas1asas200/ElecTrap_backend/blob/036787237c35ac6b2ae96ffecca6a0c285de18ec/templates/bodySelect.html#L36-L37

Also fixed `gamemode.html`.

You can write this as a template for future use if you want.